### ISSUE TITLE
If cache directory is deleted, RioFS is not creating it again unless restart RioFS

### ISSUE THREAD

**Issue by chand8** [skoobe/riofs#154]
Hi,

I was implementing RioFS on my ubuntu 16. I have configure the config file to cache data for 1 hour. For every hour, I am clearing cache in this folder by running crontjob. If the cache folder gets deleted, then RioFS is not creating the cache folder unless I restart the RioFS.

Is this a bug or I am missing any options

**Comment by wizzard** [comment]
You are right: RioFS does not re-create cache directory if it was deleted. 
This issue is in my todo list, but I can't tell you when I fix it.
